gradle/libs.versions.toml

```diff
@@ -0,0 +1,58 @@
+[versions]
+spigotApi = "1.20.2-R0.1-SNAPSHOT"
+
+adventurePlatformBukkit = "4.3.1"
+adventureTextMinimessage = "4.14.0"
+
+liteCommands = "2.8.9"
+
+okaeriConfigsYamlBukkit = "5.0.0-beta.5"
+okaeriConfigsSerdesCommons = "5.0.0-beta.5"
+okaeriConfigsSerdesBukkit = "5.0.0-beta.5"
+
+pandaUtilities = "0.5.2-alpha"
+gitCheck = "1.0.0"
+apacheCommons = "2.14.0"
+
+bStatsBukkit = "3.0.2"
+worldGuardBukkit = "7.0.9"
+
+junitJupiterApi = "5.10.0"
+junitJupiterParams = "5.10.0"
+junitJupiterEngine = "5.10.0"
+
+pluginyml = "0.6.0"
+shadowjar = "8.1.1"
+runpaper = "2.2.0"
+
+[libraries]
+spigotApi = { group = "org.spigotmc", name = "spigot-api", version.ref = "spigotApi" }
+
+adventurePlatformBukkit = { group = "net.kyori", name = "adventure-platform-bukkit", version.ref = "adventurePlatformBukkit" }
+adventureTextMinimessage = { group = "net.kyori", name = "adventure-text-minimessage", version.ref = "adventureTextMinimessage" }
+
+liteCommands = { group = "dev.rollczi.litecommands", name = "bukkit-adventure", version.ref = "liteCommands" }
+
+okaeriConfigsYamlBukkit = { group = "eu.okaeri", name = "okaeri-configs-yaml-bukkit", version.ref = "okaeriConfigsYamlBukkit" }
+okaeriConfigsSerdesCommons = { group = "eu.okaeri", name = "okaeri-configs-serdes-commons", version.ref = "okaeriConfigsSerdesCommons" }
+okaeriConfigsSerdesBukkit = { group = "eu.okaeri", name = "okaeri-configs-serdes-bukkit", version.ref = "okaeriConfigsSerdesBukkit" }
+
+pandaUtilities = { group = "org.panda-lang", name = "panda-utilities", version.ref = "pandaUtilities" }
+
+gitCheck = { group = "com.eternalcode", name = "gitcheck", version.ref = "gitCheck" }
+
+apacheCommons = { group = "commons-io", name = "commons-io", version.ref = "apacheCommons" }
+
+bStatsBukkit = { group = "org.bstats", name = "bstats-bukkit", version.ref = "bStatsBukkit" }
+
+worldGuardBukkit = { group = "com.sk89q.worldguard", name = "worldguard-bukkit", version.ref = "worldGuardBukkit" }
+
+jUnitJupiterApi = { group = "org.junit.jupiter", name = "junit-jupiter-api", version.ref = "junitJupiterApi" }
+jUnitJupiterParams = { group = "org.junit.jupiter", name = "junit-jupiter-params", version.ref = "junitJupiterParams" }
+jUnitJupiterEngine = { group = "org.junit.jupiter", name = "junit-jupiter-engine", version.ref = "junitJupiterEngine" }
+
+[plugins]
+
+pluginyml = { id = "net.minecrell.plugin-yml.bukkit", version.ref = "pluginyml" }
+shadow = { id = "com.github.johnrengelman.shadow", version.ref = "shadowjar" }
+runpaper = { id = "xyz.jpenilla.run-paper", version.ref = "runpaper" }
```

Comment:
teraz żeby prześledzić jakiekolwiek zależności per moduł to trzeba zaglądać ciągle do tego pliku. jedyne co by mogło mieć sens to wersjonowanie ale takie coś to przesada kompletnie